packages/hosted-form-v2/src/hosted-form.ts

```diff
@@ -161,7 +161,7 @@ export default class HostedForm implements HostedFormInterface {
         } catch (error) {
             // Catch form validation error because we want to trigger `onEnter`
             // irrespective of the validation result.
-            if (error.name !== 'InvalidHostedFormValueError') {
+            if (error instanceof Error && error.name !== 'InvalidHostedFormValueError') {
```

Comment:
It might be better to create typeguard for this

Reply:
It's hard to name the type of `error.name !== 'InvalidHostedFormValueError'` 😂 
I feel we can allow this one time usage.